_data-prepper/pipelines/configuration/sources/otel-logs-source.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: otel_logs_source
+parent: Sources
+grand_parent: Pipelines
+nav_order: 25
+---
+
+# otel_logs_source
+
+## Overview
+
+`otel_logs_source` is an OpenTelemetry source that follows the [OpenTelemtry Protocol Specification](https://github.com/open-telemetry/oteps/blob/master/text/0035-opentelemetry-protocol.md), receives logs from OpenTelemetry Collector, and receives `ExportLogsServiceRequest` records to process through a Data Prepper pipeline.
+
+This source supports the `OTLP/gRPC` protocol.
+{: .note}
+
+## Configuration
+
+You can configure the `otel_logs_source` source with the following options. 
+
+| Option | Type | Description |
+| :--- | :--- | :--- |
+| port | int | Represents the port that the `OTel logs source` is running on. Default value is `21892`. |
+| path | string | Represents the path for sending unframed HTTP requests. This can be used for supporting unframed gRPC with HTTP idiomatic path to a configurable path. It should start with `/` and length should be at least 1. `/opentelemetry.proto.collector.logs.v1.LogsService/Export` endpoint will be disabled for both gRPC and HTTP requests if path is configured. Path can contain `${pipelineName}` placeholder which will be replaced with pipeline name.  If the value is empty and `unframed_requests` is `true`, then the path that the source provides is `/opentelemetry.proto.collector.logs.v1.LogsService/Export`. There is no default value. | 
```

Comment:
```suggestion
| path | string | Represents the path for sending unframed HTTP requests. This can be used for supporting unframed gRPC requests with an HTTP idiomatic path that routes to a configurable path. It should start with `/` and length should be at least 1. When a path is configured, the `/opentelemetry.proto.collector.logs.v1.LogsService/Export` endpoint does not receive both gRPC and HTTP requests. Path can contain a `${pipelineName}` placeholder, which when not replaced, defaults to the current pipeline name.  If the value is empty and `unframed_requests` is `true`, then the source provides `/opentelemetry.proto.collector.logs.v1.LogsService/Export` as the path.  | 
```